pkg/tempo/config.go

```diff
@@ -117,13 +117,13 @@ func (c *Config) otelConfig() (*configmodels.Config, error) {
 
 	// todo: when we update otel collector to the latest we can just use the settings on the exporter
```

Comment:
we can drop the queue processor.  in otel latest each exporter has an independent retry queue.

oh, we can also expose the `retry_on_failure` settings as well.  right now if you can't send spans to Grafana Cloud you won't get a log message for 5 minutes b/c it will continue retrying.  At the least maybe we should default to a lower value like `60s`.

https://github.com/open-telemetry/opentelemetry-collector/blob/master/exporter/otlpexporter/README.md